main/src/ca/uwaterloo/flix/api/Bootstrap.scala

```diff
@@ -1048,13 +1048,13 @@ class Bootstrap(val projectPath: Path, apiKey: Option[String]) {
       */
     def resolveFlixDependencies(manifest: Manifest)(implicit formatter: Formatter, out: PrintStream): Result[FlixPackageManager.SecureResolution, BootstrapError] = {
```

Comment:
I guess I wanted the signature here to return a collection of errors! Or even better, use Validation.

Reply:
Ah okay. Does this at least print nicely to the terminal or no?